stm32-data-gen/src/interrupts.rs

```diff
@@ -524,7 +524,10 @@ fn valid_signals(peri: &str, chip_name: &str) -> Vec<String> {
         ("I2C", &["ER", "EV"]),
         ("I3C", &["ER", "EV", "WKUP"]),
         ("FMPI2C", &["ER", "EV"]),
-        ("TIM", &["BRK", "UP", "TRG", "COM", "CC"]),
+        (
+            "TIM",
+            &["BRK_TERR_IERR", "BRK", "UP", "TRG", "TRG_COM_DIR_IDX", "COM", "CC"],
```

Comment:
`BRK_TERR_IERR`, `TRG_COM_DIR_IDX` don't look like signal names. They're most likely a combination of multiple signals, where the signals are actually `BRK`, `TERR`, `IERR`, etc.

No idea what these are (TERR, IERR, DIR, IDX). Maybe they're something new in U3? If so we should ideally make it so they only appear in the U3 generated JSONs and not other families. 

If you want to just those signals for now that's fine too, they can be added later when someone needs them.